test/system/700-play.bats

```diff
@@ -733,3 +733,30 @@ spec:
     run_podman 125 kube play --authfile=$bogus - < $PODMAN_TMPDIR/test.yaml
     is "$output" "Error: checking authfile: stat $bogus: no such file or directory" "$command should fail with not such file"
 }
+
+@test "podman kube play with umask from containers.conf" {
+    skip_if_remote "remote does not support CONTAINERS_CONF*"
+    YAML=$PODMAN_TMPDIR/test.yaml
+
+    containersConf=$PODMAN_TMPDIR/containers.conf
+    touch $containersConf
+    cat >$containersConf <<EOF
+[containers]
+umask = "0077"
+EOF
+
+    ctr="ctr"
+    ctrInPod="ctr-pod-ctr"
+
+    run_podman create -t --restart never --name $ctr $IMAGE
```

Comment:
Please remove the `-t`; it's unnecessary, and it makes it easier to do this:
```diff
-    run_podman create -t --restart never --name $ctr $IMAGE
+    run_podman create --restart never --name $ctr $IMAGE \
+               sh -c "touch /umask-test;stat -c '%a' /umask-test"
```

Reply:
fixed. thanks for the suggestions, Ed